packages/fxa-content-server/tests/functional_smoke.js

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = [
+  //  'tests/functional/fx_browser_relier.js',
+  'tests/functional/reset_password.js',
+  'tests/functional/connect_another_device.js',
+  'tests/functional/delete_account.js',
+  //  'tests/functional/fx_fennec_v1_sign_in.js',
+  //  'tests/functional/fx_fennec_v1_sign_up.js',
+  //  'tests/functional/fx_firstrun_v2.js',
+  //  'tests/functional/fx_ios_v1_sign_in.js',
+  //  'tests/functional/fx_ios_v1_sign_up.js',
+  //  'tests/functional/oauth_reset_password.js',
+  //  'tests/functional/oauth_sign_in_token_code.js',
+  'tests/functional/oauth_sign_up.js',
```

Comment:
When this PR merges, we can add back more of these tests and work towards green on all of them.